jekyll/_cci2/reusing-config.md

```diff
@@ -459,10 +459,8 @@ commands:
 
 The following `enum` type declaration is invalid because the default is not declared in the enum list.
 
-```
-
 {% raw %}
-
+```
 yaml
```

Comment:
yaml has to be on the same line as the ticks

```yaml
version 2.1
```

Reply:
Fixed